### ISSUE TITLE
smoothness奖励问题

### ISSUE THREAD

**Issue by xiaoxiaconfidence** [IsaacZH/himloco_lab#3]
def smoothness(env: HimlocoManagerBasedRLEnv) -> torch.Tensor:
    """Penalize the rate of change of the actions using L2 squared kernel."""
    return torch.sum(torch.square(env.action_manager.action - env.action_manager.action*2 + env.pre_pre_action), dim=1)
帅哥，这个smoothness是不是有点问题，env.action_manager.action - env.action_manager.action*2，这里面没有出现pre_action,现在是pre_pre减去当前的action。
